package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.6.13",
+  "version": "0.6.14",
```

Comment:
@cameron-freshworks @andrepestana-aot @vysakh-menon-aot @argush3 

Next person to commit to this repo, please update this version to 0.7.0 ... Or d'yall think we should go to 1.0.0, now that both corrections and alterations are released?

Reply:
That should be version 7 for good marketing 😄

Reply:
1.0.0 then!

Reply:
Oops, we have a release manager! Let me ask James...